test-suite/src/lib.rs

```diff
@@ -218,17 +218,16 @@ macro_rules! generate_transaction_tests {
     };
 }
 
-#[cfg(feature = "metadata")]
 #[macro_export]
-macro_rules! generate_metadata_tests {
+macro_rules! generate_dictionary_tests {
```

Comment:
Now, `metadata` optional feature is succesfully removed. Then, we can simply move `dictionary::dictionary` integration test to default testset (`generate_store_tests`)